monai/auto3dseg/analyzer.py

```diff
@@ -799,3 +799,185 @@ def __call__(self, data):
             d[self.stats_name] = "None"
 
         return d
+
+
+class ImageHistogram(Analyzer):
+    """
+    Analyzer to compute intensity histogram.
+
+    Args:
+        image_key: the key to find image data in the callable function input (data)
+        label_key: the key to find label data in the callable function input (data)
+
+    Examples:
+
+    .. code-block:: python
+
+        import numpy as np
+        from monai.auto3dseg.analyzer import FgImageStats
+
+        input = {}
+        input['image'] = np.random.rand(1,30,30,30)
+        input['label'] = np.ones([30,30,30])
+        analyzer = FgImageStats(image_key='image', label_key='label')
+        print(analyzer(input))
+
+    """
+
+    def __init__(
+        self,
+        image_key: str,
+        stats_name: str = DataStatsKeys.IMAGE_HISTOGRAM,
+        num_of_bins: list = None,
+        hist_range: list = None,
+    ):
+
+        self.image_key = image_key
+        self.num_of_bins = num_of_bins
+        self.hist_range = hist_range
+
+        # set defaults
+        if self.num_of_bins is None:
+            self.num_of_bins: list = [100]
+        if self.hist_range is None:
+            self.hist_range: list = [-500, 500]
+
+        report_format = {"counts": None, "bin_edges": None}
+
+        super().__init__(stats_name, report_format)
+        self.update_ops(ImageStatsKeys.HISTOGRAM, SummaryOperations())
```

Comment:
I am wondering why `SummaryOperations` is used here. Can you have comments for that?